packages/cozy-harvest-lib/src/components/AccountForm/Field.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import UIField from 'cozy-ui/transpiled/react/Field'
+
+const ObfuscateChar = () => {
+  return (
+    <span style={{ fontSize: 0 }} aria-hidden="true">
+      a
+    </span>
+  )
+}
+
+/**
+ * Obfuscate a given label by inserting chars in it.
+ * The chars inserted are not visible, but they are rendered by screen readers,
```

Comment:
Are  we sure this is needed if we already transform the password field in a text field ?

I am not confortable with breaking a11y

Reply:
If this is needed (and only  if this is needed), I would welcome an `aria-label` or something similar.

Reply:
Yes it is needed. Dashlane and lastpass check the label and they seem to have a dictionary for internationalization because they match "mot de passe", "identifiant", "login" and "password".

OK I will add an aria-label

Reply:
See https://github.com/cozy/cozy-libs/pull/909/commits/2cf3d3101fdb8ab65746708fabcfae6a1b356328